env/ORION.env

```diff
@@ -132,20 +132,20 @@ elif [[ "${step}" = "anal" ]] || [[ "${step}" = "analcalc" ]]; then
 
     export NTHREADS_GSI=${nth_anal:-${nth_max}}
     [[ ${NTHREADS_GSI} -gt ${nth_max} ]] && export NTHREADS_GSI=${nth_max}
-    export APRUN_GSI="${launcher} -n ${npe_gsi:-${npe_anal}}"
+    export APRUN_GSI="${launcher} -n ${npe_gsi:-${npe_anal}} --cpus-per-task ${NTHREADS_GSI}"
```

Comment:
When I tested this, I used `--cpus-per-task=${NTHREADS_GSI}"`.   Does it work properly without the `=` sign?

Reply:
It seemed to, but I'll change it to match what you used.

Reply:
It could work without the `=`.  I didn't try.  

[slurm documentation for `--cpus-per-tasks`](https://slurm.schedmd.com/srun.html#OPT_cpus-per-task) indicates that the `=` is necessary.  Looks like `-c ` works without an `=` sign.